datasrc/network.py

```diff
@@ -585,4 +589,8 @@
 	NetMessageEx("Sv_ChangeInfoCooldown", "change-info-cooldown@netmsg.ddnet.org", [
 		NetTick("m_WaitUntil")
 	]),
+
+	NetMessageEx("Sv_MapSoundGlobal", "map-sound-global@netmsg.ddnet.org", [
+		NetIntAny("m_SoundId", 0),
```

Comment:
Was the 0 intentional? If not:
```suggestion
		NetIntAny("m_SoundId"),
```